src/cpr_sdk/vespa.py

```diff
@@ -103,9 +103,15 @@ def build_vespa_request_body(
         vespa_request_body["ranking.profile"] = "hybrid_no_closeness"
     else:
         vespa_request_body["ranking.profile"] = "hybrid"
-        vespa_request_body["input.query(query_embedding)"] = embedder.embed(
-            parameters.query_string, normalize=False, show_progress_bar=False
-        )
+        if parameters.experimental_encode_on_vespa:
+            vespa_request_body[
+                "input.query(query_embedding)"
+            ] = "embed(msmarco-distilbert-dot-v5, @query_string)"
```

Comment:
**suggestion:** So that these don't fall out-of-sync:
```suggestion
            ] = f"embed({embedder.model_name}, @query_string)"
```

This comes from: https://github.com/climatepolicyradar/cpr-sdk/blob/2d671025fe6576ac5973ee14dbaf432d4bcc7e57/src/cpr_sdk/embedding.py#L6

Reply:
that won't work here as the name used is actually set in the schema